pandas/core/frame.py

```diff
@@ -4341,27 +4341,31 @@ def last_valid_index(self):
 
     def pivot(self, index=None, columns=None, values=None):
         """
+        Return reshaped DataFrame summarized by given index / column values.
```

Comment:
I think "summarized" is misleading here, as there is no "summary" (aggregation) calculation happening. 

(I don't directly have the "better" wording in mind though. And the "reshaping" is important, as you already mention)